src/main/java/org/opensearch/timeseries/rest/handler/AbstractTimeSeriesActionHandler.java

```diff
@@ -321,13 +323,86 @@ protected void validateName(boolean indexingDryRun, ActionListener<T> listener)
             listener.onFailure(createValidationException(AbstractTimeSeriesActionHandler.INVALID_NAME_SIZE, ValidationIssueType.NAME));
             return;
         }
-        validateTimeField(indexingDryRun, listener);
+        validateTimeFieldInAllClusters(indexingDryRun, listener);
     }
 
-    protected void validateTimeField(boolean indexingDryRun, ActionListener<T> listener) {
+    protected void validateTimeFieldInAllClusters(boolean indexingDryRun, ActionListener<T> listener) {
+        List<String> wildcardClusterIndices = config.getIndices().stream().filter(idx -> idx.startsWith("*:")).toList();
+        List<String> nonWildcardClusterIndices = config.getIndices().stream().filter(idx -> !idx.startsWith("*:")).toList();
+
+        Map<String, List<String>> clusterIndicesMap = new HashMap<>();
+
+        // if no wildcard cluster indices found, fallback to typical validation
+        if (wildcardClusterIndices.isEmpty()) {
+            if (!nonWildcardClusterIndices.isEmpty()) {
+                HashMap<String, List<String>> nonWildcardMap = CrossClusterConfigUtils
+                    .separateClusterIndexes(nonWildcardClusterIndices, clusterService);
+                clusterIndicesMap.putAll(nonWildcardMap);
+                validateTimeField(clusterIndicesMap, indexingDryRun, listener);
+            } else {
+                listener
+                    .onFailure(
+                        new ValidationException(
+                            "No indices specified for time field validation.",
+                            ValidationIssueType.TIMEFIELD_FIELD,
+                            configValidationAspect
+                        )
+                    );
+            }
+            return;
+        }
+
+        // if there are wildcard cluster indices, check for remote clusters. throw exception if none configured
+        RemoteClusterService remoteClusterService = transportService.getRemoteClusterService();
+        Set<String> remoteClusters = remoteClusterService.getRegisteredRemoteClusterNames();
```

Comment:
Can we wrap the call in a try/catch where we throw listener.onFailure in case of exception?